package.json

```diff
@@ -36,6 +37,8 @@
     "class-validator": "^0.14.0",
     "convict": "^6.2.4",
     "dotenv": "^16.3.1",
+    "lodash": "^4.17.21",
+    "node-fetch": "^3.3.2",
```

Comment:
You have removed node-fetch from imports, but have added node-fetch package, why?
Also do we need lodash here?

Reply:
`node-fetch` is not really required as I've started using native Fetch API.

I'll double check on `lodash` — I had some issue with my dev env, and it was showing `loadash` as a missing dependency. It might not be the case anymore.

Reply:
`lodash` entry removed